guides/source/asset_pipeline.md

```diff
@@ -724,7 +737,7 @@ If you have other manifests or individual stylesheets and JavaScript files to
 include, you can add them to the `precompile` array in `config/initializers/assets.rb`:
 
 ```ruby
-Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += ['admin.js', 'admin.css', 'swfObject.js']
```

Comment:
Really nit-picky but why did you change the style here ? This is not consistent with other code snippets and the code generated by Rails by default. :blush: